### PR TITLE
ko-KR: Translations for OpenRCT2/OpenRCT2#7961

### PR DESCRIPTION
**Pull request by telk5093**
Translations for OpenRCT2/OpenRCT2#7961

### CODE REVIEW

File: objects/rct2/station/rct2.station.abstract.json [hunk 1]
```diff
@@ -17,7 +17,8 @@
     ],
     "strings": {
         "name": {
-            "en-GB": "Abstract"
+            "en-GB": "Abstract",
+	    "ko-KR": "추상적"
```
Comment (Gymnasiast) [inline objects/rct2/station/rct2.station.abstract.json]: Could you fix the alignment of this line? It's also off in some of the other files.

Comment (telk5093) [reply objects/rct2/station/rct2.station.abstract.json]: Done. Sorry ;)

File: objects/official/station/openrct2.station.noplatformnoentrance.json [hunk 2]
```diff
@@ -12,7 +12,8 @@
     "strings": {
         "name": {
             "en-GB": "None, and no platform",
-            "nl-NL": "Geen, ook geen perron"
+            "nl-NL": "Geen, ook geen perron",
+            "ko-KR": "감추기, 승강장도 감추기",
```
Comment (Gymnasiast) [inline objects/official/station/openrct2.station.noplatformnoentrance.json]: JSON unfortunately does not allow trailing commas, so you'll have to remove it from this line.

Comment (telk5093) [reply objects/official/station/openrct2.station.noplatformnoentrance.json]: Oops, sorry 👍

**Comment by telk5093** [comment]
@Gymnasiast I think it is not merged into OpenRCT2 itself projects. (checked in e2a91e5)

**Comment by Gymnasiast** [comment]
@telk5093 I know, I have to do that manually.

**Pull request merged**